content/project/board-election-process.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: simplepage
+title: "Board election process"
```

Comment:
IMHO it should be rather converted to JEP and stored there

Reply:
@timja @daniel-beck wdyt?

Reply:
I'm not sure if the minutes bit really belongs here but I think the rest of the content goes along well to supplement the board page, with what the board does and the actual election process...